docs/how-to/move-ipfs-installation.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Move an IPFS installation
+---
+
+# Move an IPFS installation
+
+This page walks through how to move your IPFS installation from one system to another. This process is cross-platform compatible and is especially useful for readers who are changing operating systems. The process is to grab the `.ipfs` folder from the _home_ directory of the _donor_ system, and copy it over to _home_ directory on the receiving system.
+
+This process is not a backup procedure; do not treat it as such. Many things can go wrong with this process, most of which relate to _peer IDs_. Since we are duplicating an `.ipfs` installation folder, both IPFS clients have the same peer ID. This issue is ok as long as you delete the old _donor_ installation once the copy is complete. Having two IPFS installations with the same peer ID causes substantial problems and could result in loss of data.
+
+## Move installation
+
+The easiest way to move your IFPS installation is to grab the `.ipfs` folder, and move it to another destination.
+
+### Linux and macOS
+
+1. Stop any IPFS daemons, services, or applications from running.
+1. In a terminal, move to where your IPFS repository is stored, likely your _home_ folder:
+
+    ```bash
+    cd ~/
+    ```
+
+1. Make a copy of the `.ipfs` directory:
+
+    ```bash
+    cp --recursive --verbose .ipfs ipfs-from-linux
+
+    > '.ipfs' -> 'ipfs-from-linux'
+    > '.ipfs/datastore_spec' -> 'ipfs-backup/data'
+    > ...
+    ```
+
+1. You now have a copy of your IPFS repository within the `ipfs-backup`.
```

Comment:
The copy is actually in `ipfs-from-linux` ?

Reply:
Good catch. I originally had separate sections for MacOS and Linux, but figured I'd just throw them in together. Now the instructions say to move  `.ipfs` to `ipfs-backup`.